staff/joel-molero/mailchimp/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mailchimp</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="topbar">
+            <div class="mailchimplogo">
+                <p><img src="../mailchimp/images/img/main_logo.png" alt=""></p>
```

Comment:
-Alt should not be empty

-If you add to img a Width=100%; attribute in css you will have an adaptive logo